packages/periphery/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@uma/periphery",
+  "version": "1.0.0",
+  "description": "UMA Core Peripheral Scripts",
+  "dependencies": {
+    "@ethersproject/units": "^5.0.3",
+    "@uma/common": "^2.6.0",
+    "@uma/core": "^2.7.0",
+    "@uma/financial-templates-lib": "^2.6.1",
+    "dotenv": "^6.2.0",
+    "hardhat": "^2.4.3",
+    "minimist": "^1.2.0",
+    "winston": "^3.2.1",
+    "web3": "^1.3.5"
+  },
+  "devDependencies": {},
+  "homepage": "https://umaproject.org",
+  "license": "AGPL-3.0-or-later",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.com/",
+    "access": "public"
+  },
```

Comment:
If you want to set the package to private (so it isn't published to npm), you can replace this field with the following:
```
  "private": true,
```